config/adminjs.ts

```diff
@@ -0,0 +1,56 @@
+import { AdminJSProviderConfig } from '@adminjs/adonis'
+import componentLoader from '../app/admin/component_loader.js'
+import authProvider from '../app/admin/auth.js'
+import { academicCalendarResource } from '../app/admin/resources/academic_calendars.js'
+import { buildingResource, campusResource } from '../app/admin/resources/buildings.js'
+
+const adminjsConfig: AdminJSProviderConfig = {
+  adapter: {
+    enabled: true,
+  },
+  adminjs: {
+    rootPath: '/admin',
+    loginPath: '/admin/login',
+    logoutPath: '/admin/logout',
+    componentLoader,
+    resources: [academicCalendarResource, buildingResource, campusResource],
```

Comment:
if you want to have all "resources" in one folder, then you can drop this list to `index.ts` there?